docs/cli/dnx-migration.md

```diff
@@ -0,0 +1,152 @@
+
+CLI comes packaged in two main ways, as was specified in the [intro document](overview.md#installation):
+
+1. Native installers for a given platform
+2. Install script for other situations (like CI servers)
+
+Given this, the DNVM install features are not needed. But what about the runtime selection features? 
+
+You reference a runtime in your `project.json` by adding a package of a certain version to your dependencies. With this change
+your application will be able to use the new runtime bits. Getting these bits to your machine is the same as with the CLI: 
+you install the runtime via one of the native installers it supports or via its install script. 
+
+### Different commands
+If you were using DNX, you used some commands from one of its three parts (DNX, DNU or DNVM). With the CLI, some of these
+commands change, some are not available and some are same but have slightly different semantics. 
+
+The table below shows the mapping between the DNX/DNU commands and their CLI counterparts.
+
+
+| DNX command                    	| CLI command    	| Description                                                                                                     	|
+|--------------------------------	|----------------	|-----------------------------------------------------------------------------------------------------------------	|
+| dnx run                        	| dotnet run     	| Run code from source.                                                                                           	|
+| dnu build                      	| dotnet build   	| Build an IL binary of your code.                                                                                	|
+| dnu pack                       	| dotnet pack    	| Package up a NuGet package of your code.                                                                        	|
+| dnx [command] (e.g. "dnx web") 	| N/A\*          	| In DNX world, run a command as defined in the project.json.                                                     	|
+| dnu install                    	| N/A\*          	| In the DNX world, install a package as a dependency.                                                            	|
+| dnu restore                    	| dotnet restore 	| Restore dependencies specified in your project.json.                                                            	|
+| dnu publish                    	| dotnet publish 	| Publish your application for deployment in one of the three form (portable, portable w/ native and standalone). 	|
+| dnu wrap                       	| N/A\*          	| In DNX world, wrap a project.json in csproj.                                                                    	|
+| dnu commands                   	| N/A\*          	| In DNX world, manage the globally installed commands.                                                           	|
+
+(\*) - these features are not supported in the CLI by design. 
+
+## Features that are not supported
```

Comment:
Might consider renaming this to be specific about DNX features that aren't supported.